web_src/js/modules/toast.ts

```diff
@@ -77,9 +77,12 @@ function showToast(message: string, level: Intent, {gravity, position, duration,
   });
 
   toast.showToast();
-  toast.toastElement.querySelector('.toast-close').addEventListener('click', () => toast.hideToast());
-  toast.toastElement.setAttribute('data-toast-unique-key', duplicateKey);
-  (toast.toastElement as ToastifyElement)._giteaToastifyInstance = toast;
+  if (toast.toastElement) {
```

Comment:
`toast.toastElement` must exist

Reply:
I've remove the if clause because we have a type assertion before that already eliminates this null.